views/index.hbs

```diff
@@ -33,7 +33,7 @@
             </div>
         </form>
 
-        <div id="results"></div>
+        <div id="results">{{> prs}}</div>
```

Comment:
Sorry, been trying to look into up but can't find it...mind sharing with me what the `>` does here? I know it's important but just wondering :)